src/BaseMultiValueControl.ts

```diff
@@ -76,18 +78,19 @@ export class BaseMultiValueControl {
      */
     public invalidate(): void {
         if (!this._flushing) {
+            this.setMessage("getting current value");
```

Comment:
Do we need this?

Reply:
Its so that if getting the formservice & field value takes too long we get better customer complaints about what the actual issue is.

Reply:
The goal is if any part of the loading process gets slow or crashes its trivial to know which part that is.